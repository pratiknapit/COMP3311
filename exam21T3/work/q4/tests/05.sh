psql property -c 'select id,ptype,address(id) from properties order by id limit 10;'
