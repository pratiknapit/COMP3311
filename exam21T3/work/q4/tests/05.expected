  id   |   ptype   |                address                 
-------+-----------+----------------------------------------
 45678 | House     | 10 Moira Crescent, Coogee 2034
 45679 | Apartment | 16/97 Paul Street, Bondi Junction 2022
 45680 | House     | 66 Jellicoe Avenue, Kingsford 2032
 45681 | Apartment | 11/41 Raleigh Street, Coogee 2034
 45682 | House     | 94 Burrows Road, Alexandria 2015
 45683 | Apartment | 9/7 Forsyth Street, Kingsford 2032
 45684 | Townhouse | 50 Court Avenue, Kingsford 2032
 45685 | House     | 191 Botany Road, Alexandria 2015
 45686 | Apartment | 20/28 Quail Street, Coogee 2034
 45687 | House     | 19 Holland Street, Alexandria 2015
(10 rows)
